pages/chat.php

```diff
@@ -27,9 +27,17 @@ function chat() {
 
 	// Validate post and submit to DB
     $chat_max_post_length = System::CHAT_MAX_POST_LENGTH;
+    //Increase chat length limit for seal users & staff members
+    if($player->forbidden_seal || $player->staff_level) {
+        if($player->staff_level) {
+            $chat_max_post_length = System::$premium_benefits[2]['chat_post_size'];
+        }
+        else {
+            $chat_max_post_length = System::$premium_benefits[$player->forbidden_seal['level']]['chat_post_size'];
+        }
+    }
```

Comment:
```suggestion
    //Increase chat length limit for seal users & staff members
    if($player->staff_level) {
        $chat_max_post_length = System::$premium_benefits[2]['chat_post_size'];
    }
    else if($player->forbidden_seal) {
        $chat_max_post_length = System::$premium_benefits[$player->forbidden_seal['level']]['chat_post_size'];
    }
```

Can simplify this logic